generated/nidaqmx/_task_modules/channels/ai_channel.py

```diff
@@ -398,6 +397,56 @@ def ai_accel_charge_sensitivity_units(self):
             self._handle, self._name)
         check_for_error(error_code)
 
+    @property
+    def ai_accel_db_ref(self):
```

Comment:
I think the `_db_ref` rename is good. 

However, I think we should consider deprecating the old names rather than just removing them. One way to do this is by adding hardcoded property definitions to the bottom of `ai_channel.py.mako`. 

```
%for old_property, new_property in DEPRECATED_PROPERTIES:
    @deprecation.deprecated
    @property
    def ${old_property}(self):
        return self.${new_property}

    @deprecation.deprecated
    @${old_property}.setter
    def ${old_property}(self, val):
        self.${new_property} = val

    @deprecation.deprecated
    @${old_property}.deleter
    def ${old_property}(self): 
        del self.${new_property}

%endfor
```

Another way is to directly instantiate properties (assuming this works). I'm not sure how to make this generate a `DeprecationWarning`.
```
%for old_property, new_property in DEPRECATED_PROPERTIES:
    ${old_property} = property(${new_property}.fset, ${new_property}.fget, ${new_property}.fdel)
%endfor    
```

Yet another way is to override `__getattr__` etc. and do it at run time (generate `DeprecationWarning` and redirect to the new proeprty). This is more complicated, but it hides the old properties from documentation, PyCharm, VS Code, etc.

@zhindes @maxxboehme @mshafer-NI Any thoughts?

Reply:
I like your recommendation of adding a `deprecation` dependency and keeping those old mangled names.